docs/blog/_posts/2019-04-10-14th-dotty-milestone-release.md

```diff
@@ -0,0 +1,200 @@
+
+```scala
+val copier = new Copier
+copier.print(copier.scan())
+```
+
+The motivation for this change is to promote aggregation over inheritance. In OOP languages it is easy to define inheritance but the above example would be tricky if you follow the aggregation route. One would need to implement proxy methods in the `Copier` to delegate to those of the `Scanner` and the `Printer`.
+
+With the `export` feature, making the `Copier` behave as the `Printer` and the `Scanner` became much more ergonomic. Also, note the fine-grained control over which methods are exposed in cases of the possible method collision, as shown with the `status` method example.
+
+For more information, please read more in the [documentation](http://dotty.epfl.ch/docs/reference/other-new-features/export.html).
+
+
+## An immutable array type
+A new type, `scala.IArray[T]`, is added, which is an immutable version of the `Array` type. Its implementation deserves a special attention, as it uses the new Dotty features in an elegant way (the below is an abstract from the corresponding [commit](https://github.com/lampepfl/dotty/commit/af2a0e66eb4b1204eac5dcb1d979486b92ef93d7#diff-156dc405d9f228bbc0fe406dfba63f65):
+
+```scala
+opaque type IArray[T] = Array[T]
+
+object IArray {
+
+  implied arrayOps {
+    inline def (arr: IArray[T]) apply[T] (n: Int): T = (arr: Array[T]).apply(n)
+    inline def (arr: IArray[T]) length[T] : Int = (arr: Array[T]).length
+  }
+  def apply[T: ClassTag](xs: T*): IArray[T] = Array(xs: _*)
+  /*...*/
+}
+```
+
+Essentially, the above defines a wrapper around the ordinary `Array` which exposes only its `apply` (to get an element by index) and `length` methods:
+
+- `opaque type IArray[T]` defines a type which is known to be an `Array`, but this information is known only in its companion object `IArray`. To the rest of the world, this information is not available.
+- The `implied arrayOps` implied instance defines the extension methods that expose the operations available on the `IArray` type.
+- The extension methods, `apply` and `length`, delegate to these of the `Array` type. These methods are inlined which means the performance footprint is the same as that of the original `Array` methods.
+- Because the methods are `inline` and because the `IArray` type is `opaque`, we need the `(arr: Array[T])` casts. `IArray` is known to be equal to `Array` only inside the `IArray` object and the inlining will cause the extension methods' bodies to appear outside the `IArray` object when these methods are called.
+
+## Creator Applications
+This new language feature is a generalisation of the ability to construct case classes without the `new` keyword:
+
+```scala
+class StringBuilder(s: String) {
+  def this() = this("")
+}
+StringBuilder("abc")  // same as new StringBuilder("")

```

Comment:
Comment should be `// same as new StringBuilder("abc")`

Reply:
Fixed